### PR TITLE
Update linux-nginx.md

### LINKED ISSUE #22141: Recommend increasing keepalive_requests in nginx config
Our sample nginx configuration in https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/linux-nginx?view=aspnetcore-5.0#configure-nginx does not configure a keepalive_requests to be higher than its default value of 1000: http://nginx.org/en/docs/http/ngx_http_core_module.html#keepalive_requests

In older versions of nginx, this defaulted to 100 leading to even worse perf. We should consider increasing this (to Int32.MaxValue?) in our sample config. It might be worth consulting with nginx experts about why the default is lower. But in [YARP](https://github.com/microsoft/reverse-proxy) for example, there is no per-connection request limit by default.

@sebastienros @Tratcher 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: dcdd2c35-bf2b-2673-d150-303a8aaa54d2
* Version Independent ID: 815fe034-c9fc-aee3-a8b5-f2e860b30302
* Content: [Host ASP.NET Core on Linux with Nginx](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/linux-nginx?view=aspnetcore-5.0)
* Content Source: [aspnetcore/host-and-deploy/linux-nginx.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/host-and-deploy/linux-nginx.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-hostdeploy**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #22141

[Internal review URL](https://review.learn.microsoft.com/en-us/aspnet/core/host-and-deploy/linux-nginx?view=aspnetcore-8.0&branch=pr-en-us-28483&tabs=linux-ubuntu#increase-keepalive_requests)

### CODE REVIEW

File: aspnetcore/host-and-deploy/linux-nginx.md [hunk 1]
```diff
@@ -542,12 +546,12 @@ This header prevents most browsers from MIME-sniffing a response away from the d
 1. Save the file.
 1. Restart Nginx.
 
+## Additional resources
+
```
Comment (Rick-Anderson) [inline aspnetcore/host-and-deploy/linux-nginx.md:550]:
```suggestion
```

File: aspnetcore/host-and-deploy/linux-nginx.md [hunk 2]
```diff
@@ -546,8 +550,6 @@ This header prevents most browsers from MIME-sniffing a response away from the d
 
 After upgrading the shared framework on the server, restart the ASP.NET Core apps hosted by the server.
```
Comment (Rick-Anderson) [inline aspnetcore/host-and-deploy/linux-nginx.md:552]:
```suggestion

## Additional resources
```

File: aspnetcore/host-and-deploy/linux-nginx.md [hunk 3]
```diff
@@ -547,7 +551,6 @@ This header prevents most browsers from MIME-sniffing a response away from the d
...
 
 ## Additional resources
```
Comment (Rick-Anderson) [inline aspnetcore/host-and-deploy/linux-nginx.md:553]:
```suggestion
## Additional resources

```

**Comment by Rick-Anderson** [comment]
@Tratcher  @halter73  here's the response I got back
_there are no know memory leaks in nginx. You can safely keep a connection open for thousands of requests. Even millions._

See https://github.com/dotnet/AspNetCore.Docs/issues/22141 for more info

**Review by halter73 — APPROVED** [review]

**Pull request merged**